src/printer.js

```diff
@@ -2916,6 +2936,54 @@ function printNode(path, options, print) {
 
       return path.call(print, "name");
     }
+    case "match": {
+      if (!isMinVersion(options.phpVersion, "8.0")) {
+        throw new Error("PHP Version must be > 8.0 for match expression");
```

Comment:
This line is still lacking coverage. I think we can drop the entire check though, because we're only using version checks when we have version-dependent output (e.g. short array syntax).